MvpUtility/EventHandling/MvpStats.cs

```diff
@@ -210,10 +220,8 @@ internal void OnRoundEnd(RoundEndedEventArgs ev)
             }
             else
             {
-                foreach (Player player in Player.List)
-                {
-                    player.ShowHint(hintToShow, plugin.Config.HintDisplayLimit);
-                }
+                Map.ShowHint(hintToShow, plugin.Config.HintDisplayLimit);
+                MVPMessage = hintToShow;
```

Comment:
If this works, sure, I'll test it later

Reply:
Wrong place to put this logic, once the string is genrated, then you return.

Reply:
string hintToShow = outputHint.ToString();

After this, you can return. However, this is a temporarily solution, I would rather just move the "generate hintstring" to custom function. However, the current logic is still incorrect.